Dockerfile

```
@@ -15,10 +15,7 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
   # dependencies for building Python packages
   build-essential \
   # psycopg2 dependencies
-  libpq-dev
-  # WeasyPrint dependencies \
-  weasyprint \
-  python3-cffi python3-brotli libpango-1.0-0 libpangoft2-1.0-0
+  libpq-dev \
```

now that you have removed the weasyprint stuff line 18 doesnt need a \ anymore.
```suggestion
  libpq-dev
```